client/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "client",
```

Comment:
A lot of these files came from the create-react-app Open source, which is just a quick start to React. It gave us a lot of set-up to make our lives easier. You can read into create-react-app.